.github/workflows/test-codeql.yml

```
@@ -0,0 +1,64 @@
+jobs:
+  test-codeql:
+    name: Test CodeQL Setup
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    steps:
+    - name: Checkout repository
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)